FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -237,6 +246,110 @@ - (void)callListeners:(NSString *)key config:(NSDictionary *)config {
   }
 }
 
+- (void)setCustomSignals:(nullable NSDictionary<NSString *, NSObject *> *)customSignals
```

Comment:
Any reason the API needs to support a nullable dictionary and an empty dictionary? Can  this be `nonnull`?

Reply:
You're right, we initially thought allowing nulls, but a non-null dictionary actually provides better type safety and simpler logic. I've updated the code accordingly.